packages/graphql/src/services/accounts.ts

```diff
@@ -53,3 +62,47 @@ export async function login({ email, password }: LoginArgs, ctx: Context) {
 
   return user.id
 }
+
+export async function loginParticipant(
+  { username, password }: LoginParticipantArgs,
+  ctx: Context
+) {
+  const participant = await ctx.prisma.participant.findUnique({
+    where: { pseudonym: username },
+  })
+
+  if (!participant) return null
+
+  const isLoginValid = await bcrypt.compare(password, participant.password)
+
+  if (!isLoginValid) return null
+
+  // TODO: Add back once DB table is updated
+  // ctx.prisma.participant.update({
+  //   where: { id: participant.id },
+  //   data: { lastLoginAt: new Date() },
+  // })
+
+  const jwt = JWT.sign(
+    {
+      sub: participant.id,
+      role: UserRole.PARTICIPANT,
+    },
+    // TODO: use structured configuration approach
+    process.env.APP_SECRET as string,
+    {
+      algorithm: 'HS256',
+      expiresIn: '6d',
+    }
+  )
+
+  ctx.res.cookie('user_token', jwt, {
```

Comment:
```suggestion
  ctx.res.cookie('participant_token', jwt, {
```

`user_token` is reserved for normal users and admins -> we have two so that admin users can also impersonate a participant without being logged out from the admin view